### DISCUSSION TITLE
I have two project. How can I only allow users to log in to one projectand disable the other project

### DISCUSSION THREAD

**Discussion by wqdygkd** [zitadel/zitadel#5393] [Q&A] [answered]
I have two project. How can I only allow users to log in to one projectand disable the other project

**Comment by livio-a** [comment, marked as answer]
Hey @wqdygkd 

You can restrict the login to users with granted authorization: https://zitadel.com/docs/concepts/usecases/saas#project-settings
![image](https://user-images.githubusercontent.com/9405495/223634745-a9f96788-b5dc-4703-a4da-e4d690738f51.png)

Would that solve your case?

**Reply by wqdygkd** [reply]
yes ，thankyou

in addition, my application has readwrite, and readonly permissions. How do I set permissions in zitadel, How does zitadel roles relate to application permissions

**Reply by livio-a** [reply]
ZITADEL only handles roles and not permissions (RBAC). Your application will need to map the roles to specific permissions, e.g. in your case there could be a `Admin` (readwrite) and `User` (readonly) role.
You have various options to get the role(s) of the current user, e.g. through token introspection on your API, assert them automatically into the id_token for your SPA, etc.
This blog post may also be interesting: https://zitadel.com/blog/authorization-with-role-based-access-control

**Reply by wqdygkd** [reply]
how can i set in minio with consoleAdmin role ,i use keycloak work well, i want change to zitadel

this is keycloak setting: https://github.com/minio/minio/blob/master/docs/sts/keycloak.md
